hedera-node/configuration/dev/application.properties

```diff
@@ -3,3 +3,4 @@ ledger.autoRenewPeriod.minDuration=10
 ledger.autoRenewPeriod.maxDuration=1000000000
 upgrade.artifacts.path=data/upgrade
 contracts.chainId=298
+contracts.allowAutoAssociations=true
```

Comment:
Is it possible to specify this differently, maybe directly in the test? Getting approval for this may take some time.

Reply:
I think it's possible by overriding in the tests, we have to do it for each test that needs it, but also it says that only your approval will be sufficient.